extensions/wikia/AdEngine/js/utils/env.js

```diff
@@ -0,0 +1,13 @@
+define('ext.wikia.adEngine.utils.env', [
+	'wikia.window'
+], function (win) {
+	'use strict';
+
+	function isDevEnvironment() {
+		return /.+wikia-dev\.com$/.test(win.location.host);
```

Comment:
thanks @macbre and @nandy-andy . The best code reviews are the ones that challenges the idea 👍 In our use case the domain matters not environment so it's either `wikia-dev.com` or `wikia.com` - no need to support stagings as well. I checked what Bucky does and I think we should go down that road.